changelog.d/20230620_163516_faraz.maqsood_faraz_add_temporary_patch_for_cache_program_command_issue_for_discovery.md

```diff
@@ -0,0 +1,2 @@
+- 💥[Improvement] Add a patch for this commit (https://github.com/openedx/edx-platform/pull/32506/commits/b3e9f0cbd5e44f25a42a7cbf4929b71ef122b697)
```

Comment:
In the changelog, try to add information that is actually useful to the end user. In this case, I suggest:

```
- [Bugfix] Make it possible to cache programs for a single site, instead of crashing when running `manage.py lms cache_programs` for all sites. (by @Faraz32123)
```

Note that this is not a breaking change, so we should not add "💥".